index.html

```diff
@@ -31,16 +34,187 @@ <h1>Melanie Cascante</h1>
     </section>
 
     <div class="box">
-        <a href="#">
-            <i class="Start_collab"></i>
+        <a href="#" class="button1">
+            <i class="Start_collab"></i>Start Collaboration</a>
```

Comment:
- [Optional] Please check this and correct the color of the button text to match the figma template.
### Your design
![1](https://user-images.githubusercontent.com/94551484/213788024-f0440962-4f92-4045-955e-e19555946f7f.png)
### Figma
![f1](https://user-images.githubusercontent.com/94551484/213788044-958d359e-cd63-48a3-90be-1537ac308c91.png)